v3/generator/client/client.go

```diff
@@ -40,12 +40,7 @@ func Generate(doc libopenapi.Document, path, packageName string) error {
 	`, packageName))
 
 	for _, s := range r.Model.Servers {
-		api, err := extractAPIName(s)
-		if err != nil {
-			return err
-		}
-
-		if api != "api" {
+		if !strings.Contains(s.URL, "api") {
```

Comment:
the previous check seemed more accurate to me. Why remove it?

Reply:
Since now, the spec has been updated to only one server containing production, we can rely on it.
And get rid of too much check on the spec, already checked internally.
https://github.com/exoscale/egoscale/pull/631/commits/cbb5d518f6893e398fed9637751cf29e3da4632a